src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -189,7 +190,7 @@ private String determineHdFromNormal(final String urlNormal) {
     }
 
     private void filmSuchen_old(final String urlSendung, final String thema, final String titel,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)